clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/cache/Keys.groovy

```diff
@@ -85,12 +86,18 @@ class Keys {
             detail     : names.detail
         ]
         break
-      case Namespace.HTTP_HEALTH_CHECKS.ns:
+      case Namespace.HEALTH_CHECKS.ns:
```

Comment:
Seems like it might be more future-proof to include the 'type' of the health check as a key segment to avoid potential naming collisions.